core/templates/pages/exploration-player-page/templates/display-solution-modal.controller.spec.ts

```diff
@@ -98,7 +98,7 @@ describe('Display Solution Modal Controller', function() {
     var recordedVoiceovers = recordedVoiceoversObjectFactory.createEmpty();
     card = stateCardObjectFactory.createNewCard(
       'Card 1', 'Content html', 'Interaction text', interaction,
-      recordedVoiceovers, 'content_id');
+      recordedVoiceovers, null, 'content_id');
```

Comment:
Here's an example of the comment I made for StateCardObjectFactory.ts. The value null is wrong, right? Please use a valid one.

(Note -- there may be other cases of this in other files, please check all instances of createNewCard().)